lib/note-info/index.tsx

```diff
@@ -87,6 +88,23 @@ export class NoteInfo extends Component<Props> {
               </span>
             </p>
           )}
+          <p className="note-info-item">
+            <span className="note-info-item-text">
+              <span className="note-info-name">Created</span>
+              <br />
+              <span className="note-info-detail">
```

Comment:
We need a "theme-color-fg-dim" class on "note-info-detail"

Reply:
Ah thank you. That must have come from your earlier change and I missed it when I rebased. I have it updated now.